File: app/src/main/java/ru/nehodov/weatherforecast/settings/SettingsActivity.java
package ru.nehodov.weatherforecast.settings;

import androidx.appcompat.app.AppCompatActivity;
import androidx.appcompat.widget.Toolbar;

import android.os.Bundle;

import dagger.hilt.android.AndroidEntryPoint;
import ru.nehodov.weatherforecast.R;

@AndroidEntryPoint
public class SettingsActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
        Toolbar toolbar = findViewById(R.id.settingsToolbar);
        toolbar.setNavigationOnClickListener((view) -> onBackPressed());
    }
}

File: app/src/main/java/ru/nehodov/weatherforecast/settings/SettingsActivity.kt
package ru.nehodov.weatherforecast.settings

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.appcompat.widget.Toolbar
import ru.nehodov.weatherforecast.R

class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)
        val toolbar = findViewById<Toolbar>(R.id.settingsToolbar)
        toolbar.setNavigationOnClickListener { onBackPressed() }
    }
}